### ISSUE TITLE
Allow CTRL+Click to Select multiple elements from a library

### ISSUE THREAD

**Issue by vilobion** [richardwilkes/gcs#581]
SHIFT+Click works as expected for selecting consecutive elements, but CTRL+Clicking non-consecutive elements doesn't.

**Comment by richardwilkes** [comment]
No, I will not change from Alt (or Option, depending on your platform) to Ctrl for this.

Using Ctrl conflicts with some platform's user input management, so is best avoided.

**Comment by richardwilkes** [comment]
So, it turns out that the conflicts that I originally changed this from ctrl to alt for was being caused by some automation software that some folks had installed. When they turned it off, the conflict went away. So... I've decided to go ahead and put this back since I don't see a strong need to support automation software that is getting in the way of normal operation.
